pkg/controller/kubernetes/util/configz/configz.go

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
please import `configz.go` from `k8s.io/component-base/configz`

Reply:
`k8s.io/component-base/configz` only has configz.go file after 1.19.x version. Currently, the used functions have been merged into `pkg/controller/kubernetes/cmd/controller-manager/app/serve.go`. And after replacing `k8s.io/kubernetes/pkg/controller` with `k8s.io/controller-manager`, this file will not be directly used, so it will be deleted after upgrading.